File: Coverage/webapp/src/app/shared/dto/area-coverage-gaps.ts
import { Employe } from './employe';
import { DayCoverageGaps } from './day-coverage-gaps';

export class AreaCoverageGaps {
  area: string;
  singleEmploye: boolean;
  employes: Employe[];
  gaps: DayCoverageGaps[];
}

File: Coverage/Dto/AreaCoverageGaps.cs
using System.Collections.Generic;

namespace Seac.Coverage.Dto
{
    public class AreaCoverageGaps
    {
        public string Area { get; set; }
        public bool SingleEmploye { get; set; }
        public List<EmployeBaseDto> Employes { get; }
        public List<DayCoverageGaps> Gaps { get; } = new List<DayCoverageGaps>();

        public AreaCoverageGaps(string area, IList<EmployeBaseDto> employes, bool singleEmploye)
        {
            Area = area;
            SingleEmploye = singleEmploye;
            Employes = employes as List<EmployeBaseDto>;
        }

    }
}